src/components/TopMenu.tsx

```diff
@@ -178,7 +178,9 @@ function MenuList() {
   const { t } = useTranslation()
   const { chainId } = useActiveWeb3React()
   const gaugesAreActive = areGaugesActive(chainId)
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   const { pathname } = useLocation()
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```

Comment:
Lost types or incompatible w\ latest version. Will have to prioritize router package upgrade.